.github/workflows/runtests.yml

```diff
@@ -23,16 +23,13 @@ jobs:
           python-version: '3.11'
       - run: python -m pip install --upgrade pip
 
-      - run: pip install mpmath flake8 flake8-comprehensions ruff pytest
+      - run: pip install mpmath pytest pre-commit
```

Comment:
Is pytest needed here?

Reply:
Oh, maybe it's needed for `bin/test quality`.

Ideally we would fold those checks into `pre-commit` as well in future.

Reply:
That's a good idea. I'll implement it. I know that some do not work/are to slow (e.g. mypy) so it might not be possible for everything.